docs/Tutorials/Migrating-from-Orion-to-Tessera.md

```diff
@@ -7,16 +7,20 @@ description: Migrate your Orion configuration and data to Tessera.
 
 ## Migration Process
 
+We recommend use of Tessera as a drop-in replacement private transaction manager in place of Orion.
+As Tessera supports the same endpoints and functionality, nothing needs to be changed in your Besu deployment to migrate.
+However, short downtime is required to migrate the private state database, transaction manager configuration and associated key files.
```

Comment:
Maybe "short downtime" is misleading of the actual requirements on Ops teams to perform the migration.

Maybe we could precise that
-> all private nodes (Besu+Priv Tx Manager) in the network need to go down at the same time to perform the migration
-> non-private nodes can remain up
-> [optional] Tessera and Orion can not communicate with each other (thus the requirement for network interuption)

Reply:
Looks good, thanks

Maybe in the steps you also want to highlight it:

```md
1. Shut down all privacy-enabled nodes in the network (non-privacy-enabled nodes can remain active)
...
1. Restart  all privacy-enabled nodes in the network
```